internal/schema/ize-spec.json

```
@@ -182,129 +209,167 @@
+                    "type": "string",
+                    "description": "timeout"
                 },
                 "docker_registry"  : {
-                    "type": "string"
+                    "type": "string",
+                    "description": "docker registry"
                 }
             },
+            "description": "app configuration",
             "additionalProperties": false
         },
         "ecs": {
             "id": "#/definitions/ecs",
             "type": "object",
             "properties": {
                 "path": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "path to ecs app folder"
                 },
                 "unsafe": {
-                    "type": "boolean"
+                    "type": "boolean",
+                    "description": "unsafe mode (fast deploy)"
                 },
                 "image" : {
-                    "type": "string"
+                    "type": "string",
+                    "description": "docker image"
                 },
                 "cluster"   : {
-                    "type": "string"
+                    "type": "string",
+                    "description": "ECS cluster"
                 },
-                "task_definition_revision"   : {
-                    "type": "string"
+                "task_definition_revision" : {
+                    "type": "string",
+                    "description": "task definition revision"
                 },
-                "timeout"  : {
-                    "type": "string"
+                "timeout" : {
+                    "type": "string",
+                    "description": "timeout"
                 },
                 "docker_registry"  : {
-                    "type": "string"
+                    "type": "string",
+                    "description": "docker registry"
                 }
             },
+            "description": "ecs app configuration",
             "additionalProperties": false
         },
         "serverless": {
             "id": "#/definitions/serverless",
             "type": "object",
             "properties": {
                 "file": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "path to serverless file"
                 },
                 "node_version": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "node version"
                 },
                 "create_domain" : {
-                    "type": "boolean"
+                    "type": "boolean",
+                    "description": "create domain"
                 },
                 "path"   : {
-                    "type": "string"
+                    "type": "string",
+                    "description": "path to serverless app folder"
                 },
                 "sls_node_module_cache_mount"   : {
```

@psihachina can we rename
```suggestion
                "sls_node_modules_cache_mount"   : {
```